presto-iceberg/pom.xml

```diff
@@ -505,6 +504,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Why is this change needed?

Reply:
This is needed to make use of HiveTestUtils.getProperty() method in IcebergQueryRunner.